community/planning/features.md

```diff
@@ -18,3 +18,4 @@ community meeting.
 | Feature | Description |
 | ------- | ----------- |
 | [Location Sapling]({% link community/planning/location_sapling.md %}) | A front-end for Grid Location functionality |
+| [Batch Processor]({% link community/planning/batch_processor.md %}) | A daemon processor for submitting batches |
```

Comment:
Should rename the file - batch_submitter.md

Reply:
renamed to batch_submitter.md